flink-runtime/src/test/java/org/apache/flink/runtime/state/StateBackendMigrationTestBase.java

```diff
@@ -1052,7 +1061,7 @@ public void testBroadcastStateKeySerializerReconfiguration() throws Exception {
                         IntSerializer.INSTANCE));
     }
 
-    @Test
+    @TestTemplate
     public void testBroadcastStateRegistrationFailsIfNewValueSerializerIsIncompatible() {
```

Comment:
```suggestion
    void testBroadcastStateRegistrationFailsIfNewValueSerializerIsIncompatible() {
```
could be package private